supernode/server/router.go

```diff
@@ -48,6 +48,8 @@ func initRoute(s *Server) *mux.Router {
 		{Method: http.MethodGet, Path: "/peer/piece/suc", HandlerFunc: s.reportPiece},
 		{Method: http.MethodGet, Path: "/peer/service/down", HandlerFunc: s.reportServiceDown},
 		{Method: http.MethodGet, Path: "/peer/piece/error", HandlerFunc: s.reportPieceError},
+		{Method: http.MethodPost, Path: "/peer/network", HandlerFunc: s.getP2PNetworkInfo},
```

Comment:
Why is this POST method named `getP2PNetworkInfo`? Is it should be `GET` method?

Reply:
fix it